Flaky E2E test: ScaleOutFrom1To3

The CI fails with:

```
 === RUN   TestScaling/ScaleOutFrom1To3/ScaleOutFrom1To3/verify_member_list
    e2e_test.go:173: Found unpromoted learner after scaling completed: member etcd-scaleoutfrom1to3-2 (17278970040244508223)
--- FAIL: TestScaling (80.35s)
    --- PASS: TestScaling/ScaleInFrom3To1 (40.07s)
        --- PASS: TestScaling/ScaleInFrom3To1/ScaleInFrom3To1 (40.07s)
            --- PASS: TestScaling/ScaleInFrom3To1/ScaleInFrom3To1/scale_to_1 (10.01s)
            --- PASS: TestScaling/ScaleInFrom3To1/ScaleInFrom3To1/verify_member_list (0.03s)
    --- FAIL: TestScaling/ScaleOutFrom1To3 (40.28s)
        --- FAIL: TestScaling/ScaleOutFrom1To3/ScaleOutFrom1To3 (40.28s)
            --- PASS: TestScaling/ScaleOutFrom1To3/ScaleOutFrom1To3/scale_to_3 (30.16s)
            --- FAIL: TestScaling/ScaleOutFrom1To3/ScaleOutFrom1To3/verify_member_list (0.03s) 
```

Refer to: https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/etcd-io_etcd-operator/297/pull-etcd-operator-test-e2e/2028548466015211520#

--- a/test/e2e/e2e_test.go
+++ b/test/e2e/e2e_test.go
@@ -188,6 +188,7 @@ func TestScaling(t *testing.T) {
 				waitForNoLearners(t, c, podName, tc.expectedMembers, 3*time.Minute)
 
 				// Verify controller promoted all learners to voting members
+				ml = getEtcdMemberListPB(t, c, podName)
 				for _, m := range ml.Members {
 					if m.IsLearner {
 						t.Errorf("Found unpromoted learner after scaling completed: member %s (%d)", m.Name, m.ID)

